spec/controllers/idv/review_controller_spec.rb

```diff
@@ -32,10 +20,15 @@
     )
     idv_session.profile_confirmation = true
     idv_session.vendor_phone_confirmation = true
-    idv_session.applicant = user_attrs
+    idv_session.applicant = applicant.with_indifferent_access
```

Comment:
Curious about this change: Is it a simple hash in the real-world? Was the previous implementation not working anymore with the revisions introduced here?

(May be related to / puntable to LG-7123)

Reply:
It's a `HashWithIndifferentAccess` in the real world when it's accessed from the password confirm controller

```
pry(#<Api::Verify::PasswordConfirmController>)> user_session['idv']['applicant'].class
=> ActiveSupport::HashWithIndifferentAccess
```

Reply:
Thanks for checking! Sounds good then.